src/components/ImmutableDevices/ImmutableDevices.js

```diff
@@ -16,6 +16,9 @@ const ImmutableDevices = ({
   filterConfig,
   hideFilters,
   activeFiltersConfig,
+  tableActions,
+  onRefresh,
+  actionsConfig,
```

Comment:
Why not add a `...props` and use it below with the `InventoryTable`?

```js
<InventoryTable
   showTags
   ...
   { ...props }
/>
```

Later on if we see that certain attributes are used all over the place with the ImmutableDevices component we can add them here with proper default.

Reply:
Done.